virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -59,7 +59,7 @@ endif::[]
 
 * To deploy {VirtProductName} in a disconnected environment, you must xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[configure Operator Lifecycle Manager on restricted networks].
 
-* When using a disconnected cluster on a restricted network, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. Using a proxy allows the cluster to fetch the {VirtProductName} Operator.
+* When using a disconnected cluster on a restricted network, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. Using a proxy allows the cluster to fetch the {VirtProductName} Operator. A proxy can also be used if the cluster is partially connected. See xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for more information.
```

Comment:
"When using a partially connected cluster like a cluster in a restricted network, you must"

and drop the whole part "A proxy can also be used if the cluster is partially connected. See xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for more information." as this is pretty redundant to line 60.

Reply:
Per our GChat we agreed to:

With limited internet connectivity, you can xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. If you are using a restricted network with no internet connectivity, you must xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Configure Operator Lifecycle Manager for restricted networks].

If this looks good, @fabiand please re-approve. Thanks!